src/js/plugin/PluginDispatcherProxy.js

```diff
@@ -0,0 +1,36 @@
+import PluginDispatcher from "./external/PluginDispatcher";
+
+export default class PluginDispatcherProxy {
```

Comment:
Curious question:
Is it possible to derive from the PluginDispatcher like this?
```JavaScript
export default class PluginDispatcherProxy extends PluginDispatcher {
```

Then you could get rif of the simple forwarding of `register`, `unregister`, `waitFor` and so..